app/controllers/api/v1/application_users_controller.rb

```diff
@@ -135,6 +135,32 @@ def index
 #end
 
   ###############################################################
+  # username
+  ###############################################################
+
+  api :GET, '/application_users/username/:username',
+            'Gets a single User with the specified username.'
+  description <<-EOS
+      All users of OpenStax have an associated User object.  This endpoint
+      allows querying additional data on a user when all that is known is the
+      User's username.
+
+      Admins (for Accounts only) are identified by the is_administrator boolean.
+      Some additional user information can be found in associations, such as
+      email addresses in ContactInfos and the password hash in Identity.
+
+      Users have the following String attributes:
+      username, first_name, last_name, full_name, title
+
+    #{json_schema(Api::V1::UserSearchRepresenter, include: :readable)}
+  EOS
+  def username
+    OSU::AccessPolicy.require_action_allowed!(:search, current_api_user, User)
+    outputs = SearchUsers.call("username:#{params[:username]}", {exact: true}).outputs
```

Comment:
Looks like I can't use standard_read, since accounts are at version 2.2.4.  That standard_read's signature is: `standard_read(model_klass, id, represent_with=nil)`, and it can only lookup using `model_klass.find`

It's only a few lines difference though to do the same work in the method, I'll go that route